{
  "seleniumLog": [],
  "checkPoints": [],
  "errorTrace": "java.lang.RuntimeException: com.qmetry.qaf.automation.testng.DataProviderException: Error while fetching data from resources/testdata.csv\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:111)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:509)\r\n\tat org.testng.internal.Invoker.handleParameters(Invoker.java:1293)\r\n\tat org.testng.internal.Invoker.createParameters(Invoker.java:1020)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1110)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:781)\r\n\tat org.testng.TestRunner.run(TestRunner.java:635)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1133)\r\n\tat org.testng.TestNG.run(TestNG.java:1104)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\nCaused by: com.qmetry.qaf.automation.testng.DataProviderException: Error while fetching data from resources/testdata.csv\r\n\tat com.qmetry.qaf.automation.util.CSVUtil.getCSVDataAsMap(CSVUtil.java:186)\r\n\tat com.qmetry.qaf.automation.testng.dataprovider.QAFInetrceptableDataProvider.getData(QAFInetrceptableDataProvider.java:325)\r\n\tat com.qmetry.qaf.automation.testng.dataprovider.QAFInetrceptableDataProvider.interceptedDataProvider(QAFInetrceptableDataProvider.java:117)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:55)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)\r\n\t... 22 more\r\nCaused by: java.lang.NullPointerException\r\n\tat com.qmetry.qaf.automation.util.CSVUtil.getCSVDataAsMap(CSVUtil.java:167)\r\n\t... 31 more\r\n"
}